dotcom-rendering/src/web/components/HeaderTopBar.importable.tsx

```diff
@@ -31,7 +32,7 @@ const topBarStyles = css`
 		padding-left: 15px;
 	}
 	${from.desktop} {
-		height: 2.5rem;
+		height: 2.1875rem;
```

Comment:
Please use pixels! They are [turned into `rem` as part of our build](https://github.com/guardian/dotcom-rendering/blob/5d7d7b516be013134f847ddcbb3bef31258992ed/dotcom-rendering/babel.config.js#L8).